gulpfile.js

```diff
@@ -383,4 +383,7 @@ gulp.task('e2e-test', gulp.series(clean, setupE2e, gulp.parallel('build-bundle-p
 gulp.task(bundleToStdout);
 gulp.task('bundle', gulpBundle.bind(null, false)); // used for just concatenating pre-built files with no build step
 
+// build task for reviewers, runs test-coverage, serves, without watching
+gulp.task('review-start', gulp.series(clean, lint, gulp.parallel('build-bundle-dev', testCoverage)));
```

Comment:
i'm probably missing something obvious, but where is the "serve" part? does testCoverage actually execute a serve?

Reply:
Good catch, I must have had an old "watch" running in another terminal when I was testing this. I need to refactor the "serve" call a bit since it cleans, etc. Will get it fixed up.

Reply:
I added back the "watch" instead of getting to a refactor of the core commands. Removing the watch element was a nicety, not a requirement.